priv/static/common.css

```diff
@@ -0,0 +1,49 @@
+/* This file contains design tokens used in core Gleam projects */
+:root {
+  /* Greys */
+  --col-black: #000000;
+  --col-grey-950: #151515;
+  --col-grey-900: #1e1e1e;
+  --col-grey-800: #2f2f2f;
+  --col-grey-100: #f5f5f5;
+  --col-azure-900: #202431;
+  --col-azure-800: #292d3e;
+  --col-azure-600: #51597b;
+  --col-azure-500: #7e818b;
+  --col-sepia-450: #b1a894;
+  --col-sepia-400: #cac0a9;
+  --col-sepia-300: #e3d8be;
+  --col-sepia-100: #fffbe8;
+  --col-sepia-050: #fefefc;
+
+  /* Colors */
+  --col-faff-pink-800: #584355;
+  --col-faff-pink-300: #ffaff3;
```

Comment:
The names of the colours should not change. We can use modifiers for shades and tints. If a colour is a named colour already then we don't add any extra modifier to the variable name as it implies that it is not that colour.

If there's a --col-faff-pink-800 and a --col-faff-pink-300 it's unclear which is faff pink.

Reply:
Agreed in case of the faff pink, maybe I misunderstood your intention. So you would prefer that the menthol, caramel etc do not have numbers?

Reply:
On a side note, I think it would've been useful to keep numbers for colors to indicate that they belong to the same "family".

For example we could later have lighter and dimmer variants of the green "menthol" color for things like border, text, background for "success"-type messages/buttons. This is also the reason I had chosen generic hue names (in this case: green), as the name "methol" already implies a bright color

Reply:
it's using the current branding colors now 🚀